### DISCUSSION TITLE
Field Conditions Filter by current user roles

### DISCUSSION THREAD

**Discussion by DamienMrtl** [directus/directus#13625] [Feature Requests]
It would be great if we could hide or show a field depending on the role of the user. 

I can't achieve this by removing the read permission on the field because then it breaks relations display.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
